src/paths/widest_paths.c

```diff
@@ -0,0 +1,747 @@
+/* -*- mode: C -*-  */
+/* vim:set ts=4 sw=4 sts=4 et: */
+/*
+   IGraph library.
+   Copyright (C) 2005-2021 The igraph development team
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+   02110-1301 USA
+
+*/
+
+
+#include "igraph_paths.h"
+
+#include "igraph_adjlist.h"
+#include "igraph_interface.h"
+#include "igraph_memory.h"
+#include "igraph_qsort.h"
+
+#include "core/indheap.h"
+#include "core/interruption.h"
+
+/**
+ * \function igraph_get_widest_paths
+ * \brief Widest paths from a single vertex
```

Comment:
There's a `.` missing :)
```suggestion
 * \brief Widest paths from a single vertex.
```

Reply:
Done :)